src/Sources/Code/Module.cpp

```diff
@@ -535,19 +670,27 @@ void CALLBACK MyDispatchProc(SIMCONNECT_RECV* pData, DWORD cbData, void* pContex
 				std::string prefix = "MF.SimVars.Set.";
 				str = str.substr(prefix.length());
 #if _DEBUG
-				std::cout << "MobiFlight[" << client->Name.c_str() << "]: Executing Code: " << str.c_str() << std::endl;				
+				std::cout << "MobiFlight[" << client->Name.c_str() << "]: Executing Code: " << str.c_str() << std::endl;
 #endif
 				execute_calculator_code(str.c_str(), 0, nullptr, nullptr);
 				break;
 			}
-		
+
 			std::shared_ptr<std::string> m_str = std::make_shared<std::string>(str);
-			
+
 			if (m_str.get()->find("MF.SimVars.Add.") != std::string::npos) {
 				std::string prefix = "MF.SimVars.Add.";
 				str = m_str.get()->substr(prefix.length());
-				RegisterSimVar(str, client);
-				std::cout << "MobiFlight[" << client->Name.c_str() << "]: Received SimVar to register: " << str.c_str() << std::endl;				
+				RegisterSimVar(str, client, false);
```

Comment:
in context of my earlier comments, this could be renamed to a proper RegisterFloatSimVar